Modules/CheckClassVersion.cmake

```
@@ -135,6 +135,14 @@ function(check_class_version)
       list(APPEND CMD_ENV "${ev}=$ENV{${ev}}")
     endif()
   endforeach()
+  if (TARGET ${dictname}_dict)
+    set(LD_PATH_FOR_DICT "$<JOIN:$<TARGET_PROPERTY:${dictname}_dict,LINK_DIRECTORIES>,:>")
+    if (DEFINED ENV{LD_LIBRARY_PATH})
+      string(APPEND LD_PATH_FOR_DICT ":$ENV{LD_LIBRARY_PATH}")
+    endif()
```

Can I suggest:
```suggestion
    string(JOIN ":" LD_PATH_FOR_DICT "${LD_PATH_FOR_DICT}" $ENV{LD_LIBRARY_PATH})
```
which should work regardless of whether the environment variable is unset or empty.